src/main/java/com/sequenceiq/cloudbreak/service/stack/handler/StackDeleteCompleteHandler.java

```diff
@@ -57,7 +58,9 @@ private void updateStackFields(Stack stack) {
         Cluster cluster = stack.getCluster();
         if (cluster != null) {
             cluster.setName(terminatedName);
+            cluster.setBlueprint(null);
```

Comment:
They will be deleted at usage generation when the stack will be deleted with its resources. Probably the user does not want to delete the recipes individually.